lib/testrail.rb

```diff
@@ -7,79 +7,102 @@ module Gridium
 	class TestRail
 		ENV_ERROR = "Environment Variable not set!"
 
-    #TestRail Statuses
-    PASSED = 1
-    BLOCKED = 2
-    UNTESTED = 3
-    RETEST = 4
-    FAILED = 5
-
 		def initialize
       if Gridium.config.testrail
         @url = ENV['GRIDIUM_TR_URL'].empty? || ENV['GRIDIUM_TR_URL'].nil? ? ENV_ERROR : ENV['GRIDIUM_TR_URL'] + '/index.php?/api/v2/'
         @user = ENV['GRIDIUM_TR_USER'].empty? || ENV['GRIDIUM_TR_USER'].nil? ? ENV_ERROR : ENV['GRIDIUM_TR_USER']
         @password = ENV['GRIDIUM_TR_PW'].empty? || ENV['GRIDIUM_TR_PW'].nil? ? ENV_ERROR : ENV['GRIDIUM_TR_PW']
         @pid = ENV['GRIDIUM_TR_PID'].empty? || ENV['GRIDIUM_TR_PID'].nil? ? ENV_ERROR : ENV['GRIDIUM_TR_PID']
-				@testcase_ids = Array.new
-				@testcase_infos = Array.new
+				@config = {:pass => 1, :blocked => 2, :untested => 3, :retest => 4, :fail => 5}
+				@retry_attempts = 5
+				@time_between_retries = 3
```

Comment:
These two i-vars should be constants, they don't need any type of interesting initialization. Throw 'em up around line 8